### PR TITLE
Task 684

### PR DESCRIPTION
**Pull request by islamTaha12**

### CODE REVIEW

File: functional_testing/0_orchestrator/README.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+## 0-Orchestrator Testsuite
```
Comment (john-kheir) [inline functional_testing/0_orchestrator/README.md]: Please remove this read me file .. it's already integrated in the orchestrator repo

File: functional_testing/0_orchestrator/test_suite/orchestrator_objects/orchestrator_apis/__init__.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        except HTTPError as e:
+            if e.response.status_code == 440:
+                self.orchasterator_driver.refresh_jwt()
+                wrapper(self, *args, **kwargs)
+            else:
+                return e.response
+
+    return wrapper
+
+def catch_exception_decoration_return(func):
```
Comment (john-kheir) [inline functional_testing/0_orchestrator/test_suite/orchestrator_objects/orchestrator_apis/__init__.py]: why we can't use just the .._return decorator?

File: functional_testing/0_orchestrator/test_suite/orchestrator_objects/orchestrator_apis/bridges_apis.py [hunk 3]
```diff
@@ -0,0 +1,49 @@
+from orchestrator_objects.orchestrator_apis import *
```
Comment (john-kheir) [inline functional_testing/0_orchestrator/test_suite/orchestrator_objects/orchestrator_apis/bridges_apis.py]: import star is usually not a good idea

File: functional_testing/0_orchestrator/test_suite/orchestrator_objects/orchestrator_apis/storagepools_apis.py [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    def post_storagepools_storagepoolname_filesystems(self, node_id, storagepoolname, **kwargs):
+        data = {"name": self.random_string(),
+                "quota": random.randint(0, 10)}
+        data = self.update_default_data(default_data=data, new_data=kwargs)
+        response = self.orchestrator_client.nodes.CreateFilesystem(nodeid=node_id, storagepoolname=storagepoolname,
+                                                                   data=data)
+        return response, data
+
+    @catch_exception_decoration
+    def get_storagepools_storagepoolname_filesystems_filesystemname(self, nodeid, storagepoolname, filesystemname):
```
Comment (john-kheir) [inline functional_testing/0_orchestrator/test_suite/orchestrator_objects/orchestrator_apis/storagepools_apis.py]: long function names and long lines as well

File: functional_testing/0_orchestrator/test_suite/testcases/basic_tests/test02_bridges_apis.py [hunk 5]
```diff
@@ -0,0 +1,122 @@
+import time
+from testcases.testcases_base import TestcasesBase
+
+<<<<<<< HEAD:functional_testing/0_orchestrator/test_suite/testcases/basic_tests/test02_bridges_apis.py
```
Comment (john-kheir) [inline functional_testing/0_orchestrator/test_suite/testcases/basic_tests/test02_bridges_apis.py]: ?? :dancer:

File: functional_testing/0_orchestrator/test_suite/orchestrator_objects/orchestrator_apis/__init__.py (hunk 2 above)
Comment (islamTaha12) [inline functional_testing/0_orchestrator/test_suite/orchestrator_objects/orchestrator_apis/__init__.py]: return are differents

File: functional_testing/0_orchestrator/test_suite/orchestrator_objects/orchestrator_apis/storagepools_apis.py (hunk 4 above)
Comment (islamTaha12) [inline functional_testing/0_orchestrator/test_suite/orchestrator_objects/orchestrator_apis/storagepools_apis.py]: changed by Dina but not merged yet

File: functional_testing/0_orchestrator/test_suite/testcases/basic_tests/test02_bridges_apis.py (hunk 5 above)
Comment (islamTaha12) [inline functional_testing/0_orchestrator/test_suite/testcases/basic_tests/test02_bridges_apis.py]: solved

File: functional_testing/0_orchestrator/README.md (hunk 1 above)
Comment (islamTaha12) [inline functional_testing/0_orchestrator/README.md]: Link please

Comment (john-kheir) [inline functional_testing/0_orchestrator/README.md]: https://github.com/zero-os/0-orchestrator/blob/master/tests/Grid_API_Testing/README.md

File: functional_testing/0_orchestrator/test_suite/orchestrator_objects/orchestrator_apis/bridges_apis.py (hunk 3 above)
Comment (islamTaha12) [inline functional_testing/0_orchestrator/test_suite/orchestrator_objects/orchestrator_apis/bridges_apis.py]: but why :D

**Pull request merged**